src/tool_fsharp/tool_fsc.csproj

```diff
@@ -13,6 +13,8 @@
     <PackageReference Include="Microsoft.NetCore.App" Version="$(CLI_SharedFrameworkVersion)" />
     <PackageReference Include="Microsoft.FSharp.Compiler" Version="$(CLI_FSharp_Version)" />
     <PackageReference Include="CliDeps.Satellites.FSharp" Version="$(CLI_FSharp_Satellites_Version)" />
+    <PackageReference Include="System.Reflection.Metadata" Version="1.4.2" />
```

Comment:
I'm not sure I understand these two changes.  Do you know why they are necessary?

Reply:
There's a nuget downgrade warning as error in the f# graph without it. I couldn't figure out why it's only in source build, though. This is in the patches too.